src/Dianoga/Default Config Files/Dianoga.WebP.config.disabled

```diff
@@ -46,7 +46,7 @@
 				</processor>
 			</dianogaOptimizePng>
 			<dianogaOptimizeGif>
-				<processor type="Dianoga.Optimizers.Pipelines.DianogaWebP.WebPOptimizer, Dianoga" patch:before="processor[1]">
+				<processor type="Dianoga.Optimizers.Pipelines.DianogaWebP.GifWebPOptimizer, Dianoga" patch:before="processor[1]">
```

Comment:
I think we could simplify this by modifying `WebPOptimizer` and adding a check for `args.InputStream.Extension == "gif"`

Reply:
I see your point about having 1 single optimizer, but I'm not a fan of hardcoding functionality. I have pushed another version where I disable the resizing via configuration of the processor in the pipeline. What do you think?

Reply:
Great idea